### PR TITLE
docs(dev guide): structural-directives - ts minor edits, dart first version

### PR DESCRIPTION
**Pull request by chalin**
- copyedits and other minor edits for TS
- a first version of the chapter for Dart

### CODE REVIEW

**Comment by chalin** [comment]
Ready for review and merge.
@filipesilva : can you review the TS file?
@thso : the Dart version is pretty much just the TS version but with Dart examples.

File: public/docs/ts/latest/guide/structural-directives.jade [hunk 1]
```diff
@@ -163,7 +166,7 @@ figure.image-display
   We can confirm these effects by wrapping the middle "hip" of the phrase "Hip! Hip! Hooray!" within a `<template>` tag.
 +makeExample('structural-directives/ts/app/structural-directives.component.html', 'template-tag')(format=".")
 :marked
-  The display is a 'Hip!' short of perfect enthusiasm. The DOM effects are different when Angular is control.
+  The display is a 'Hip! Hooray!', short of perfect enthusiasm. The DOM effects are different when Angular is control.
```
Comment (zoechi) [inline public/docs/ts/latest/guide/structural-directives.jade]: :+1:

**Comment by thso** [comment]
Review this now.

File: public/docs/ts/latest/guide/structural-directives.jade (hunk 1 above)
Comment (thso) [inline public/docs/ts/latest/guide/structural-directives.jade]: Think there is an issue with end of the sentence. Do we mean: 'when Angular IS in control'?

**Comment by thso** [comment]
LGTM

File: public/docs/ts/latest/guide/structural-directives.jade (hunk 1 above)
Comment (chalin) [inline public/docs/ts/latest/guide/structural-directives.jade]: Good catch. Fixed.

**Pull request merged**